Doyeon-k-fanshawe/group2_bootcamp
# Adding personal information JS script

## What does the PR do?
* It creates the description for Abram

## How do you check this PR?
* Pull down the branch
*  See added details under Abram's profile

## Screenshots 
None.
